tests/common.rs

```diff
@@ -3,13 +3,20 @@
 extern crate fantoccini;
 extern crate futures_util;
 
-use fantoccini::{error, Client};
+use fantoccini::error;
 
 use std::future::Future;
 use std::net::{IpAddr, Ipv4Addr, SocketAddr};
 use std::path::PathBuf;
 use warp::Filter;
 
+#[cfg(feature = "rustls-tls")]
+pub type Client = fantoccini::RustlsClient;
+#[cfg(all(feature = "openssl-tls", not(feature = "rustls-tls")))]
+pub type Client = fantoccini::OpenSslClient;
```

Comment:
This won't work. We should run all the tests with both clients, not just with one or the other depending on features.

Reply:
This is still relevant I believe -- we can't provide these aliases.